llvm/lib/MC/RepoObjectWriter.cpp

```diff
@@ -562,6 +562,7 @@ RepoObjectWriter::toPstoreLinkage(GlobalValue::LinkageTypes L) {
     return pstore::repo::linkage_type::external;
   case GlobalValue::LinkOnceAnyLinkage:
   case GlobalValue::LinkOnceODRLinkage:
+  case GlobalValue::WeakAnyLinkage:
```

Comment:
Add a comment to indicate that this is a short-term workaround. 

 case GlobalValue::WeakAnyLinkage: // TODO: Implement as linkonce for now to allow LLVM to build correctly. Implement support for weak linkage in the future.

Reply:
Thanks Ben, added the comment!